modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/Relationships.tsx

```diff
@@ -174,7 +174,6 @@ export default function Relationships({
 				}
 				else {
 					deleteRelationship(itemData.id);
-					setTimeout(() => window.location.reload(), 1500);
```

Comment:
Hey @larissacribeiro, removing this line will generate a new bug, because when the deletion is successful, the page will not reload to show the updated table. So what we can do is move this line of code inside the `deleteRelationship` function inside the `try` statement after `openToast` function.

Reply:
I see, sorry about that! I've updated it right now